backend/test/akvo/flow_services/end_to_end_test.clj

```diff
@@ -217,13 +217,15 @@
 
       (is (= 200 (:status (test-util/get-report report-result))))
       (test-util/check-report-file-headers report-result "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
-      (is (= ["IN_PROGRESS" "FINISHED_SUCCESS"] (final-report-state-in-flow flow-report-id))))))
+      (is (= ["IN_PROGRESS" "FINISHED_SUCCESS"] (final-report-state-in-flow flow-report-id)))
+      (is (zero? (sentry-alerts-count))))))
```

Comment:
Can we assert that is always `zero?`? Does the wire mock counter resets when we do the setup?

Reply:
There is a _fixture_ that call wiremock reset for each test case, so we know that this should be zero.